libs/common/src/lib/policies/site.policy.ts

```diff
@@ -0,0 +1,55 @@
+import { Project, Site, User } from "@terramatch-microservices/database/entities";
+import { FrameworkKey } from "@terramatch-microservices/database/constants/framework";
+import { UserPermissionsPolicy } from "./user-permissions.policy";
+
+export class SitePolicy extends UserPermissionsPolicy {
+  async addRules() {
+    if (this.permissions.includes("sites-read") || this.permissions.includes("view-dashboard")) {
+      this.builder.can("read", Site);
+    }
+
+    const frameworks = this.permissions
+      .filter(name => name.startsWith("framework-"))
+      .map(name => name.substring("framework-".length) as FrameworkKey);
+    if (frameworks.length > 0) {
+      this.builder.can("read", Site, { frameworkKey: { $in: frameworks } });
```

Comment:
This can use `this.frameworks` from the super class instead of building the array itself.